ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryRegistryImpl.java

```diff
@@ -111,6 +130,14 @@ private QueryRegistryImpl(final QueryRegistryImpl original) {
         .filter(Optional::isPresent)
         .map(Optional::get)
         .collect(Collectors.toList());
+    if (!original.sandbox) {
```

Comment:
@rodesai When fixing an issue with the parametrization of a test it surfaced an issue where the registry is creating a sandbox from a sandbox. This was causing an exception. This happens only during the replacement of a persistent query. It doesn't make sense to me why thing is happening. I will look into it more tomorrow but perhaps you have some idea? 

This fix does not work it just prevents the exception but no data is produced

Reply:
I adapted it to be able to copy a validation runtime but still no data is coming though